app/controllers/payment_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PaymentController < ApplicationController
```

Comment:
actually I think we no longer need a payments controller. Instead we handle the redirecting in the sessions controller so we don't have to check for authorization here. It will be a lot easier too because the conditional statement will just check a user's logged in status.